packages/paste-core/components/combobox/src/singleselect/Combobox.tsx

```diff
@@ -166,8 +167,7 @@ const Combobox = React.forwardRef<HTMLInputElement, ComboboxProps>(
               {...getInputProps({disabled, required, ref, ...props})}
               {...(!autocomplete ? {onChange: (event: React.ChangeEvent) => event.preventDefault()} : undefined)}
               autocomplete={autocomplete}
-              aria-describedby={helpTextId}
-              aria-label={hideVisibleLabel ? labelText : null}
+              aria-describedby={helpText != null ? helpTextId : null}
```

Comment:
question: why this change?

Reply:
so it was throwing an incomplete check, which I'm guessing it was doing for the hidden label too, where we reference an id that doesn't exist in the DOM if you don't have help text. So it might be an error that would get thrown at some point in the future.

This way it'll never be a problem. It only gets a describebed by attribute if there is a helptext passed in.